debezium-core/src/test/java/io/debezium/relational/mapping/OriginalDataTypeTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.relational.mapping;
+
+import io.debezium.relational.Column;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.junit.Test;
+
+import java.sql.Types;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+/**
+ * @author Orr Ganani
+ *
+ */
+public class OriginalDataTypeTest {
+
+    private final int length = 255;
+    private final Column column = Column.editor()
+        .name("col")
+        .type("VARCHAR")
+        .jdbcType(Types.VARCHAR)
+        .length(length)
+        .create();
+
+    @Test
+    public void shouldTruncateStrings() {
```

Comment:
The name seems a bit odd?

Can you also add the `FixFor` annotation, referencing the issue's JIRA number?